docs/monitor/view-active-alarms.md

```diff
@@ -1,75 +1,75 @@
 <!--
-title: "View active health alarms"
-description: "View active alarms and their rich data to discover and resolve anomalies and performance issues across your infrastructure."
-custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/monitor/view-active-alarms.md
+title: "View active health alerts"
+description: "View active alerts and their rich data to discover and resolve anomalies and performance issues across your infrastructure."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/monitor/view-active-alerts.md
```

Comment:
@samrafalowski the URL is returning me a 404 so I did not approve for now

Reply:
Fixed this